monolithic/monolithic-layered/src/main/java/com/epam/training/microservices/monolithic/model/delivery/Delivery.java

```diff
@@ -49,8 +50,9 @@ public class Delivery {
     private Pharmacy pharmacy;
 
     /**
-     * Content of the com.epam.training.service.delivery.
+     * Content of the delivery.
      */
+    @ToString.Exclude
```

Comment:
I think, the `pharmacy` field should also be exclude from `toString` as it is already excluded from equals and hashCode. Would it be better to put `@ToString(exclude={“pharmacy”, “content”})` on top of the class?